presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -3103,4 +3105,17 @@ public FeaturesConfig setPrintEstimatedStatsFromCache(boolean printEstimatedStat
         this.printEstimatedStatsFromCache = printEstimatedStatsFromCache;
         return this;
     }
+
+    public boolean getUseNewNanDefinition()
+    {
+        return useNewNanDefinition;
+    }
+
+    @Config("use-new-nan-definition")
```

Comment:
Should we call this the new NaN definition, and by default this is false, or should we refer to the old behavior as the deprecated NaN definition, and by default that is true?

Reply:
Oh, I see the later commit that enables it.  I would have recommended this if it were disabled by default, but I think it's fine as it is.